addons/web_editor/static/tests/wysiwyg_tests.js

```diff
@@ -2529,11 +2873,70 @@ QUnit.test('checklist', function (assert) {
                         '<p>y</p>',
                 },
             },
+            {
```

Comment:
Broken test.
This test should be moved to another commit (not fixed in the current one)